src/scss/index.scss

```diff
@@ -251,3 +141,4 @@ h2 {
 }
 
 @import './themes/themes';
+@import './viewer.scss';
```

Comment:
@Gauravjeetsingh would you please add a new line at the end of file?

Reply:
fixed